cmd/nerdctl/run_linux.go

```diff
@@ -82,6 +83,19 @@ func setPlatformOptions(opts []oci.SpecOpts, cmd *cobra.Command, id string) ([]o
 		opts = append(opts, secOpts...)
 	}
 
+	lab, err := cmd.Flags().GetStringArray("label")
+	if err != nil {
+		return nil, err
+	}
+	labelMaps := strutil.ConvertKVStringsToMap(strutil.DedupeStrSlice(lab))
```

Comment:
We need to support `label-file` too, but it can be fixed in a separate PR if it is too complicated.
Please add "TODO" comment in that case.
 https://github.com/containerd/nerdctl/blob/b88e714bdc4a3603ce92ec1b5726e96c7f76eaad/cmd/nerdctl/run.go#L843